CONTRIBUTING.md

```diff
@@ -33,6 +33,10 @@ Upvote the original issue by clicking its [+😊] button and hitting 👍 (+1) i
 
 Please comment on an issue to let us know you're interested in working on something before you start the work. Not only does this avoid multiple people unexpectedly working on the same thing at the same time but it enables us to make sure everyone is clear on what should be done to implement any new functionality. It's less work for everyone, in the long run, to establish this up front.
 
+### Localisation issues
```

Comment:
This is the UK style spelling. Localization is en-us

Reply:
I’d suggest reordering. Lead with please file issues if something is not localized correctly but we cannot accept PRs as localization must flow through an internal system.